blueprints/frost-component/files/__root__/__path__/__name__.js

```diff
@@ -1,13 +1,34 @@
 import Ember from 'ember'
-import FrostEvents from 'ember-frost-core/mixins/frost-events'
-import {
-  PropTypes
-} from 'ember-prop-types'
+const { Component } = Ember
+import PropTypesMixin, { PropTypes } from 'ember-prop-types'
+import layout from '<%= templatePath %>'
 
-const {
-  Component
-} = Ember
+const <%= classifiedModuleName %> = Component.extend(PropTypesMixin, {
```

Comment:
So, are we moving away from the initializer route then? Is it better to have the explicit mixin inclusion? I thought in previous discussions your position was that prop types was one of the scenarios where initializers were the preferred mechanism.